Index: E:/WebWork/themeforest/theme-Dresscode/oc/svn/trunk/www/catalog/view/theme/dresscode/template/common/additional_links.tpl
===================================================================
--- E:/WebWork/themeforest/theme-Dresscode/oc/svn/trunk/www/catalog/view/theme/dresscode/template/common/additional_links.tpl	(revision 6)
+++ E:/WebWork/themeforest/theme-Dresscode/oc/svn/trunk/www/catalog/view/theme/dresscode/template/common/additional_links.tpl	(revision 7)
@@ -73,78 +73,4 @@
 
     </div>
 </div>
-<div id="slogan">
-            <ul>
-                <li class="links_orig_products">
-                    <a href="<?php echo $original_products; ?>">
-                        <?php echo $text_original_products; ?>
-                    </a>
-                </li>
-                <li class="links_free_shipping">
-                    <a href="<?php echo $free_shipping; ?>">
-                        <?php echo $text_free_shipping; ?>
-                    </a>
-                </li>
-                <li class="links_cash">
-                    <a href="<?php echo $cash_delivery; ?>">
-                        <?php echo $text_cash_delivery; ?>
-                    </a>
-                </li>
-                <li class="links_returns">
-                    <a href="<?php echo $day_returns; ?>">
-                        <?php echo $text_day_returns; ?>
-                    </a>
-                </li>
-            </ul>
 
-</div>
-
-<div class="container">
-    <div class="sixteen columns">
-
-        <div class="clearfix tabs">
-            <div>
-                        <a id="newproducts_activate" class="null">
-                            <table class="slider_table_style">
-                                <tr class="slider_tr_style">
-                                    <td class="slider_td_style">
-                                        <?php echo $text_new_products; ?>
-                                    </td>
-                                </tr>
-                            </table>
-
-                        </a>
-
-
-            </div>
-            <div>
-                <span>/</span>
-
-                <a id="specials_activate" class="null">
-                    <table class="slider_table_style">
-                        <tr class="slider_tr_style">
-                            <td class="slider_td_style">
-                                <?php echo $text_sale; ?>
-                            </td>
-                        </tr>
-                </table>
-
-                </a>
-
-            </div>
-            <div>
-                <span>/</span>
-                <a id="bestsellers_activate" class="null">
-                    <table class="slider_table_style">
-                        <tr class="slider_tr_style">
-                            <td class="slider_td_style">
-                                <?php echo $text_bestsellers; ?>
-                            </td>
-                        </tr>
-                    </table>
-                </a>
-            </div>
-        </div>
-
-    </div>
-</div>
Index: E:/WebWork/themeforest/theme-Dresscode/oc/svn/trunk/www/catalog/view/theme/dresscode/template/common/twitter.tpl
===================================================================
--- E:/WebWork/themeforest/theme-Dresscode/oc/svn/trunk/www/catalog/view/theme/dresscode/template/common/twitter.tpl	(revision 6)
+++ E:/WebWork/themeforest/theme-Dresscode/oc/svn/trunk/www/catalog/view/theme/dresscode/template/common/twitter.tpl	(revision 7)
@@ -26,7 +26,7 @@
             <li>
                 <div class="twitter-marker" style="padding-bottom:30px">
                     <p><a target="_blank" href="https://twitter.com/#!/<?=$twitt['user']['screen_name']?>">
-                        <?=mb_substr($twitt['text'],0,80)?></a></p>
+                        <?=$twitt['text']?></a></p>
                     <span><?=$twitt['created_at']?></span>
                 </div>
             </li>
